library/lib_window.ks

```diff
@@ -0,0 +1,26 @@
+
+
+@lazyglobal off.
+
+function create_window{
```

Comment:
Sorry, did not realize you are spammed. I did it for myself, to mark which things are already done.